doc/markup_reference.rb

```diff
@@ -0,0 +1,917 @@
+#   # ====== <i>Two words</i> in heading
+#
+# ==== Bold
+#
+# A single word may be made bold by prefixed and suffixed asterisks.
+#
+# Examples:
+#
+#   # *Word* in paragraph.
+#   # - *Word* in bullet list item.
+#   # 1. *Word* in numbered list item.
+#   # a. *Word* in lettered list item.
+#   # [*word*] *Word* in labeled list item.
+#   # ====== *Word* in heading
+#
+# Any text may be made bold via HTML tag +b+.
+#
+# Examples:
+#
+#   # <b>Two words</b> in paragraph.
+#   # - <b>Two words</b> in bullet list item.
+#   # 1. <b>Two words</b> in numbered list item.
+#   # a. <b>Two words</b> in lettered list item.
+#   # [<b>Two words</b>] <b>Two words</b> in labeled list item.
+#   # ====== <b>Two words</b> in heading
+#
+# ==== Monofont
+#
+# A single word may be made monofont -- sometimes called "typewriter font" --
+# by prefixed and suffixed plus-signs.
+#
+# Examples:
+#
+#   # +Word+ in paragraph.
+#   # - +Word+ in bullet list item.
+#   # 1. +Word+ in numbered list item.
+#   # a. +Word+ in lettered list item.
+#   # [+word+] +Word+ in labeled list item.
+#   # ====== +Word+ in heading
+#
+# Any text may be made monofont via HTML tag +tt+ or +code+.
+#
+# Examples:
+#
+#   # <tt>Two words</tt> in paragraph.
+#   # - <tt>Two words</tt> in bullet list item.
+#   # 1. <tt>Two words</tt> in numbered list item.
+#   # a. <tt>Two words</tt> in lettered list item.
+#   # [<tt>Two words</tt>] <tt>Two words</tt> in labeled list item.
+#   # ====== <tt>Two words</tt> in heading
+#
+# ==== Character Conversions
+#
+# Certain combinations of characters may be converted to special characters;
+# whether the conversion occurs depends on whether the special character
+# is available in the current encoding.
+#
+# - <tt>(c)</tt> converts to (c) (copyright character); must be lowercase.
+#
+# - <tt>(r)</tt> converts to (r) (registered trademark character); must be lowercase.
+#
+#  - <tt>'foo'</tt> converts to 'foo' (smart single-quotes).
+#    [Reviewers: This did not work; should it?]
+#
+#  - <tt>"foo"</tt> converts to "foo" (smart double-quotes).
+#
+#  - <tt>foo ... bar</tt> converts to foo ... bar (1-character ellipsis).
+#
+#  - <tt>foo -- bar</tt> converts to foo -- bar (1-character en-dash).
+#
+#  - <tt>foo --- bar</tt> converts to foo --- bar (1-character em-dash).
+#
+# ==== Links
+#
+# Certain strings in \RDoc text are converted to links.
+# Any such link may be suppressed by prefixing a backslash.
+# This section shows how to link to various
+# targets.
+#
+# [Class]
+#
+#   - On-page: <tt>DummyClass</tt> links to DummyClass.
+#   - Off-page: <tt>RDoc::Alias</tt> links to RDoc::Alias.
+#
+# [Module]
+#
+#   - On-page: <tt>DummyModule</tt> links to DummyModule.
+#   - Off-page: <tt>RDoc</tt> links to RDoc.
+#
+# [Constant]
+#
+#   - On-page: <tt>DUMMY_CONSTANT</tt> links to DUMMY_CONSTANT.
+#   - Off-page: <tt>RDoc::Text::MARKUP_FORMAT</tt> links to RDoc::Text::MARKUP_FORMAT.
+#     [Reviewers: This link did not work. Should it?]
```

Comment:
(This is me sign-posting where there are direct questions in this diff, to try making it easier to locate them. I'll quote the question text.)

> [Reviewers: This link did not work; should it?]